.drone.yml

```diff
@@ -14,9 +14,53 @@ pipeline:
   clone:
     image: plugins/git
     tags: true
-    # dont clone submodules
     recursive: false
 
+  clone-pr:
+    image: gcr.io/eminent-nation-87317/git-clone:1.1
```

Comment:
whats this the new way to clone?

Reply:
i noticed sometimes Drone doesn't take the latest commit I push so the build fails so I pinged Manjunath about this and he said this is a workaround to make sure we get the latest commit for the PR.